<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_ui</class>
 <widget class="QWidget" name="settings_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoMute Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt.qrc">
    <normaloff>:/icons/img/icon.png</normaloff>:/icons/img/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_settings">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_server_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_server_to_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add current Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_remove_server_from_list">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected Server</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_always_mute">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mute_on_all_servers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute on all Servers:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_server_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server List:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_mute_mic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mute_mic_or_speakers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Microphone or Speakers:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mute_mic">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Microphone</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_mute_speakers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mute_speakers">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>75</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speakers</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections/>
</ui>
